### PR TITLE
Feature/fu 378

### PR DESCRIPTION
**Pull request by amitmo50**

### CODE REVIEW

File: projects/fusion-ui/components/button/button.component-v3.scss [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+$fu-black-300: #565C62;
+$fu-black-500: #53575B;
+
+
+$fu-negative-500: #FF6A63;
+$fu-negative-600: #EC5D56;
+$fu-negative-700: #DA564F;
+
+
+:host.fu-style-v3 {
```
Comment (danielbrazg) [inline projects/fusion-ui/components/button/button.component-v3.scss:82]: update file like we talked about

**Review by danielbrazg — CHANGES_REQUESTED** [review]

File: projects/fusion-ui/components/button/button.component.ts [hunk 2]
```diff
@@ -21,17 +21,22 @@ import {IconData} from '@ironsource/fusion-ui/components';
...
             <ng-container *ngIf="selectedVersion$ | async as selectedVersion">
                 <fusion-icon
-                    *ngIf="isLoading"
+                    *ngIf="isLoading && (selectedVersion === 1 || selectedVersion === 2)"
                     class="icon-loading"
                     [name]="
                         selectedVersion === 1 ? {iconName: 'loading', iconVersion: 'v1'} : {iconName: 'loader-dots-v4', iconVersion: 'v2'}
                     "
                 ></fusion-icon>
+                <fusion-icon
```
[the remark this answers is not in the document]
Comment (amitmo50) [reply projects/fusion-ui/components/button/button.component.ts:30]: need to separate v3 from v1 & v2

File: projects/fusion-ui/components/button/button.component.ts [hunk 3]
```diff
@@ -21,17 +21,22 @@ import {IconData} from '@ironsource/fusion-ui/components';
             <fusion-icon *ngIf="iconName && !isLoading" class="icon" [name]="iconData"></fusion-icon>
             <ng-container *ngIf="selectedVersion$ | async as selectedVersion">
                 <fusion-icon
-                    *ngIf="isLoading"
+                    *ngIf="isLoading && (selectedVersion === 1 || selectedVersion === 2)"
```
[the remark this answers is not in the document]
Comment (amitmo50) [reply projects/fusion-ui/components/button/button.component.ts]: I change this but still need to separate between V2 & V1 and V3

File: projects/fusion-ui/components/button/button.component-v3.scss [hunk 4]
```diff
@@ -0,0 +1,657 @@
...
+    font-family:$helveticaNueMd;
+    text-align: center;
+    font-weight: 500;
+}
+
+%rotating-icon {
+    animation: rotation 1s infinite linear;
+}
+
+@keyframes rotation {
```
[the remark this answers is not in the document]
Comment (amitmo50) [reply projects/fusion-ui/components/button/button.component-v3.scss:17]: will be move when Andy merge his branch

File: projects/fusion-ui/components/button/button.component-v3.scss [hunk 5]
```diff
@@ -0,0 +1,657 @@
+@import '../../src/style/scss/vars/mixins';
```
[the remark this answers is not in the document]
Comment (amitmo50) [reply projects/fusion-ui/components/button/button.component-v3.scss:1]: will be move when Andy merge his branch

**Review by chenlevin89 — APPROVED** [review]

**Pull request merged**